spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+RSpec.describe "Movies show page", type: :feature do
+#   Story 2 - Movie Show
+
+# As a user,
+# When I visit a movie's show page.
+# I see the movie's title, creation year, and genre,
+# and a list of all its actors from youngest to oldest.
+# And I see the average age of all of the movie's actors
```

Comment:
It looks like you've got the average age in the view, but you aren't testing anywhere here that it's showing up on the page. All we'd need is something like this:
```ruby
expect(page).to have_content("Average age: 35")
```